hack/custom_certs/cr-with-custom-certs.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+  namespace: openshift-lightspeed
+spec:
+  llm:
+    providers:
+    - credentialsSecretRef:
+        name: bam-api-keys
+      type: bam
+      models:
+      - name: ibm/granite-13b-chat-v2
+      name: bam
+      url: https://bam-api.res.ibm.com
```

Comment:
@raptorsun Let us not use BAM as the example , its internal and our users cannot get access to that. Please update to watsonx,azure/openai

Reply:
ok, let's use another example